### ISSUE TITLE
Demos Not Consistently Uploading

### ISSUE THREAD

**Issue by sealscarl** [demostf/demos.tf#19]
I've had an issue ever since i started my server in May where the demos do not seem to be consistently uploading to the site. There does not seem to be any rhyme or reason as to why they don't upload, but for example... if we scrim 4 maps in a single night, 1-2 of the maps will upload, and the others will not. 

I'm not really sure where to begin trying to troubleshoot this. I always wait to change the map until the STV recording is complete message appears in chat.

**Comment by Darshil182** [comment]
yes, this. MY demos don't seem to be uploading consistently as well. I don't know why, it seems like serveme uploads fine but my old server used to upload fine as well but this new server doesnt upload demos. its random
